datasets/us/hi/med_exclusions/crawler.py

```diff
@@ -0,0 +1,64 @@
+from typing import Dict
+from rigour.mime.types import PDF
+
+from zavod import Context, helpers as h
+
+from zavod.shed.gpt import run_image_prompt
+
+prompt = """
+Extract structured data from the following page of a PDF document. Return 
+a JSON list (`providers`) in which each object represents an medical provider.
+The name of a provider might be split onto the next row within a record.
+Each object should have the following fields: `last_name`,
+`first_name`, `middle_initial`, `medicaid_provider_id`, `provider_type`,
+`exclusion_date`, `reinstatement_date`.
+Return an empty string for unset fields.
+"""
+
+
+def crawl_item(row: Dict[str, str], context: Context):
+
+    if first_name := row.pop("first_name"):
+        entity = context.make("Person")
+        entity.id = context.make_id(
+            row.get("last_name"), first_name, row.get("exclusion_date")
+        )
+        h.apply_name(
+            entity,
+            first_name=first_name,
```

Comment:
let's use the original first name value for the id, but split the first and middle names on aka, a.k.a etc when applying names. I think you can call apply_name as many times as there are name combinations